main/acle.md

```diff
@@ -1313,10 +1315,13 @@ enclose them in parentheses if they are not simple constants.
 
 ## Testing for Arm C Language Extensions
 
-`__ARM_ACLE` is defined to the version of this specification
```

Comment:
I think it would be worth keeping the description of the old versioning scheme around, because users will want to write checks which work with both the old and new schemes. The new numbers are always larger than the old ones, so that's not actually a problem, but it'll reduce confusion when old compilers emit the old version format.

Reply:
Thanks. Done